packages/suite-desktop/src-electron/libs/app-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { app, RelaunchOptions } from 'electron';
+
+const isLinux = process.platform === 'linux';
+
+export const restartApp = () => {
+    const { logger } = global;
+
+    const options: RelaunchOptions = {};
+
+    options.args = process.argv.slice(1).concat(['--relaunch']);
+
+    if (isLinux && process.env.APPIMAGE) {
```

Comment:
nit: `isLinux` condition is kind of extra here?
```suggestion
    if (process.env.APPIMAGE) {
```

Reply:
I will check that with ⬆️